vizro-core/docs/pages/user-guides/customizing-css.md

```diff
@@ -0,0 +1,330 @@
+# Customizing CSS
+
+Vizro is quite opinionated when it comes to visual formatting, but it allows for some customization if you want to
+deviate from the default settings. While certain elements, such as the layout of the navigation and components,
+are fixed, you can customize background colors, fonts, and other styles via CSS overrides.
+
+To do so, you need to:
+
+1. Add a CSS file to your `assets` folder, refer to our user guide on [adding static assets](assets.md#how-to-add-static-assets).
+2. Identify the right CSS selector for the component you want to style.
+3. Add the desired changed properties to your CSS file.
+
+
+
+## Intro to CSS
```

Comment:
```suggestion
## Introduction to CSS customization
```

Reply:
Best to have this clearly spelt out as it'll help SEO if anyone searches "customize CSS Vizro"